cronAffiliation.sh

```diff
@@ -5,9 +5,10 @@ export PYTHONPATH="$DIR/src/:$DIR/../CMSMonitoring/src/python:$PYTHONPATH"
 source "$DIR/venv/bin/activate"
 export REQUESTS_CA_BUNDLE="/etc/pki/ca-trust/extracted/openssl/ca-bundle.trust.crt"
 create=$(cat <<EOF
+import os
 from htcondor_es.AffiliationManager import AffiliationManager, AffiliationManagerException
 try:
-    AffiliationManager(recreate_older_days=1)
+    AffiliationManager(recreate_older_days=1, dir_file=os.getenv('AFFILIATION_DIR_LOCATION', AffiliationManager._AffiliationManager__DEFAULT_DIR_PATH))
```

Comment:
please document usage of all env variables such that scripts will prove proper usage message. Quite often in operations it is tedious to find them and proper documentation helps.